chanzuckerberg/idseq-web
# [Data Discovery] Make panes scroll independently

# Description

* Right pane scrolls independently with its own sidebar.
* In DD, center pane is sized given height of the window. Scrolling happen within the table.
* Also, only show scrollbar for right pane when hovering on the pane.

![Apr-08-2019 11-37-57](https://user-images.githubusercontent.com/37312125/55748198-da3e8980-59f2-11e9-997c-9a6e286a7711.gif)

# Test and Reproduce

* Load data discovery
* Open accordions on right pane or pick domain in a way that the content overflows the window
* Verify that the scroll bar is shown, on hover only, and only within the right pane
